pkg/controller/azvolume.go

```diff
@@ -328,12 +328,18 @@ func (r *ReconcileAzVolume) triggerDelete(ctx context.Context, azVolume *azdiskv
 			}
 
 			if err == nil {
+				// if the driver is uninstalling, delete azvolume CRI to let uninstallation proceed
 				if volumeDeleteRequested {
-					cloudCtx, cloudCancel := context.WithTimeout(goCtx, cloudTimeout)
-					defer cloudCancel()
+					if r.isDriverUninstall(ctx) {
+						goWorkflow.Logger().Errorf(fmt.Errorf("VolumeDeleteRequestError"), "trying to delete volume (%s) as the driver is uninstalling, deletion will be ignored", azVolume.Name)
```

Comment:
This is an expected condition and shouldn't be logged as an error. If logged, it should be done at a higher verbosity level, e.g. `goWorkflow.Logger().V(12).Infof(...)`.